### PR TITLE
feat(core): add sign in button to shared doc

### PR DESCRIPTION
**Pull request by JimmFly**
CleanShot 2024-08-23 at 12 05 12@2x

### CODE REVIEW

**Comment by JimmFly** [comment]
* **#7952** <a href="https://app.graphite.dev/github/pr/toeverything/AFFiNE/7952?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `canary`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @JimmFly and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: packages/frontend/i18n/src/resources/en.json [hunk 1]
```diff
@@ -1406,6 +1406,7 @@
   "com.affine.share-page.footer.description": "Empower your sharing with AFFiNE Cloud: One-click doc sharing",
   "com.affine.share-page.footer.get-started": "Get started for free",
   "com.affine.share-page.header.present": "Present",
+  "com.affine.share-page.header.login": "Login Or Sign Up",
```
Comment (pengx17) [inline packages/frontend/i18n/src/resources/en.json]: "Login or Sign Up"

Comment (JimmFly) [reply packages/frontend/i18n/src/resources/en.json]: by design

Comment (pengx17) [reply packages/frontend/i18n/src/resources/en.json]: I'm not sure what you mean here. Incorrect grammar in the product looks unprofessional.

Comment (JimmFly) [reply packages/frontend/i18n/src/resources/en.json]: @qpomelo Can you explain this?

**Review by pengx17 — APPROVED** [review]

File: packages/frontend/i18n/src/resources/en.json (hunk 1 above)
Comment (qpomelo) [reply packages/frontend/i18n/src/resources/en.json]: Based on our internal discussion about button styling, each word within buttons was initially capitalized. However, I suggest we refine our rules to exclude prepositions, conjunctions, and similar words from capitalization.

**Review by forehalo — APPROVED** [review]

**Review by forehalo — APPROVED** [review]

**Pull request merged**